torch/nn/modules/loss.py

```diff
@@ -933,20 +933,31 @@ class TripletMarginLoss(Module):
     convolutional feature descriptors with triplet losses`_ by
     V. Balntas, E. Riba et al.
 
+    The loss function for each sample in the mini-batch is:
+
     .. math::
-        L(a, p, n) = \frac{1}{N} \left( \sum_{i=1}^N \max \{d(a_i, p_i) - d(a_i, n_i) + {\rm margin}, 0\} \right)
+        L(a, p, n) = \max \{d(a_i, p_i) - d(a_i, n_i) + {\rm margin}, 0\}
 
     where :math:`d(x_i, y_i) = \left\lVert {\bf x}_i - {\bf y}_i \right\rVert_p`.
 
     Args:
-        anchor: anchor input tensor
-        positive: positive input tensor
-        negative: negative input tensor
-        p: the norm degree. Default: 2
+        margin (float, optional): Default: `1`.
+        p (int, optional): The norm degree for pairwise distance. Default: `2`.
+        swap (float, optional): The distance swap is described in detail in the paper
+            `Learning shallow convolutional feature descriptors with triplet losses` by
```

Comment:
Nit: does this actually link to the paper in the link below? You can check by rebuilding the docs (cd into docs/, then `make html`). The original string has a _ after the ending ` so I was wondering if that did anything.